ansible_base/lib/abstract_models/common.py

```diff
@@ -37,13 +37,10 @@ def get_url_for_object(obj, request=None):
 
 
 class CommonModel(models.Model):
-    # These are fields that should be reversed lookup as related fields.
-    # For example, an environment has related organizations so environment might specify reverse_foreign_key_fields = ['organizations']
-    # This would end up with a view like environment/1/organizations
-    reverse_foreign_key_fields = []
```

Comment:
Ok this is interesting. This is the kind of thing a changelog would be needed for. I take it that this needs a simultaneous merge in aap-gateway now, because it's pinned to devel, right?

Reply:
Yea, its not pinned. But this wouldn't require a simultaneous change (although I have one) because it just removes it off the base model. The models that inherit from it might set it as a field it just wouldn't be used anywhere.